pom.xml

```diff
@@ -54,18 +54,25 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
 
-    <opensaml.version>4.2.0</opensaml.version>
+    <opensaml.version>4.3.0</opensaml.version>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>javax.servlet</groupId>
```

Comment:
@malaporte I really think this should be a breaking release and get rid of the javax stuff entirely